pkg/kubelet/server/server.go

```diff
@@ -234,11 +236,14 @@ func NewServer(
 	server.InstallDefaultHandlers(enableCAdvisorJSONEndpoints)
 	if enableDebuggingHandlers {
 		server.InstallDebuggingHandlers(criHandler)
+	} else {
+		server.InstallDebuggingDisabledHandlers()
+	}
+	if enableProfiling {
```

Comment:
Shouldn't this be enabled only if `enableDebuggingHandlers` is true?

Reply:
done